src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h

```diff
@@ -563,3 +568,40 @@ struct xcl_firewall {
     uint64_t err_detected_level;
     uint64_t err_detected_time;
 };
+
+//
+// IOCTL_XOCL_MAILBOX_INFO
+// Get sensor info
+// Inbuffer = (not used)
+// OutBuffer = struct xcl_mailbox
+//
+#define IOCTL_XOCL_MAILBOX_INFO          CTL_CODE(FILE_DEVICE_XOCL_USER, 2112, METHOD_BUFFERED, FILE_READ_DATA)
+enum xcl_mailbox_request {
```

Comment:
This enum should be already defined in mailbox_proto.h file, which is the interface file for mailbox comm. Please include that file instead.

Reply:
Hi @arpitxilinx , mailbox_proto.h file is present in $src/runtime_src/core/pcie/driver/**linux**/include/mailbox_proto.h, so I am not sure including linux header file in windows driver is the better idea.

Also, consider "struct xcl_sensor" which is defined in mailbox_proto.h already, but windows driver is not using this structure, instead declared a new one in XoclUser_INTF.h
So, this applies to mailbox also. Please check and let me know

Reply:
@rajkumar-xilinx I absolutely agree with you that we shouldn't be including the mailbox_proto.h from linux subdirectory. What I meant is add/copy  the mailbox_proto.h from our driver repo and use it in our xrt windows usercode. This file should be treated just like XoclMgmt_INTF and XoclUser_INTF and should reside in driver/windows/include subdirectory.